### ISSUE TITLE
overfitting by using pdc loss

### ISSUE THREAD

**Issue by Ivy147** [cleardusk/3DDFA#159]
Hi，jianzhu，thank you so much for your amazing work,it helps me a lot!
But when I use your training code and 300w-lp dataset to train the model, I find that the train loss will drop from 1.0 to about 0.1,but the val loss only drop to 0.7 then begin to rise.I think it is overfitting.Have you ever had such a problem before?Or did I make any mistakes?
Please let me know if anyone knows.  Your help will be very useful to me!

**Comment by cleardusk** [comment]
pdc is easy to overfitting, but your loss gap seems to be larger than mine. You can try to tune the hyperparameters like the learning rate.
